policy-service/src/unit-tests/java/uk/gov/gchq/palisade/service/policy/ApplicationTestData.java

```diff
@@ -63,18 +66,40 @@
             .withContext(CONTEXT)
             .withUser(USER)
             .withResource(LEAF_RESOURCE);
+
     public static final PolicyResponse RESPONSE = PolicyResponse.Builder.create(REQUEST)
             .withRules(RULES);
+    public static final PolicyResponse RESPONSE_NO_RULES = PolicyResponse.Builder.create(REQUEST)
+            .withRules(new Rules<>());
 
     public static final AuditErrorMessage AUDIT_ERROR_MESSAGE = AuditErrorMessage.Builder.create()
             .withUserId(USER_ID.getId())
             .withResourceId(RESOURCE_ID)
             .withContext(CONTEXT)
             .withAttributes((new HashMap<>()))
-            .withError(new NoSuchPolicyException("Something went wrong!"));
+            .withError(new CompletionException(new NoSuchPolicyException("No rules found for the resource")));
```

Comment:
do you need the CompletionException?

Reply:
Not needed, was left over from previous assertion updates. It has now been removed.